src/site/resources/.htaccess

```diff
@@ -1,5 +1,4 @@
-#!/bin/sh
-# ----------------------------------------------------------------------------
+#
```

Comment:
_Nit_: this hidden file in a standalone folder is hard to find.

Could we have everything under the same tree? Also is `src/site/antora` the native location for Antora? If it is not, could we move it?

Reply:
I will discuss this personally and apply @ppkarwasz's suggestion to both `2.x` and `main`.